crud.py

```diff
@@ -0,0 +1,33 @@
+from sqlalchemy.orm import Session
+from models import City, Temperature
+from schemas import CityCreate, TemperatureCreate
+
+
+# CRUD для City
+def get_cities(db: Session):
+    return db.query(City).all()
+
+
+def create_city(db: Session, city: CityCreate):
+    db_city = City(name=city.name, country=city.country)
+    db.add(db_city)
+    db.commit()
+    db.refresh(db_city)
+    return db_city

```

Comment:
The `create_city` function is correctly implemented for adding a new city to the database. However, the task description requires implementing additional CRUD operations such as updating and deleting cities. Consider adding these functionalities to fully meet the task requirements.